### PR TITLE
Add note to message table how to generate debug log

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

Does what it says.

Bildschirmfoto zu 2021-11-09 07-06-54

### CODE REVIEW

**Comment by DL6ER** [comment]
Is this change intended?

Screenshot from 2021-11-09 17-00-35

**Comment by yubiuser** [comment]
No, don't know how it got there. This is for another PR that might come or might not come. Deleted it.

File: messages.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
                     </tr>
                     </thead>
                 </table>
+                <p>Note: If errors are shown, use the tab on the left to generate a debug log, which will show a more thorough Pi-hole evaluation.</p>
```
Comment (DL6ER) [inline messages.php]: Maybe include a link here?

```suggestion
                <p>Note: If errors are shown, you can <a href="debug.php">generate a debug log</a>, which will do a thorough Pi-hole evaluation.</p>
```

Comment (yubiuser) [reply messages.php]: Good idea.

**Review by DL6ER — APPROVED** [review]
Screenshot from 2021-11-21 21-33-05

**Pull request merged**